### ISSUE TITLE
[PAGE ISSUE]: 'Flutter for Xamarin.Forms developers'

### ISSUE THREAD

**Issue by Paul-N** [flutter/website#8921]
### Page URL

https://docs.flutter.dev/get-started/flutter-for/xamarin-forms-devs/

### Page source

https://github.com/flutter/website/tree/main/src/get-started/flutter-for/xamarin-forms-devs.md

### Describe the problem

Wrong source code for https://docs.flutter.dev/get-started/flutter-for/xamarin-forms-devs#how-do-i-show-the-progress-for-a-long-running-task
same as in the #8920 
Corrected code below

### Expected fix

```
import 'dart:async';
import 'dart:convert';

import 'package:flutter/material.dart';
import 'package:http/http.dart' as http;

void main() {
  runApp(const SampleApp());
}

class SampleApp extends StatelessWidget {
  const SampleApp({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      title: 'Sample App',
      home: SampleAppPage(),
    );
  }
}

class SampleAppPage extends StatefulWidget {
  const SampleAppPage({super.key});

  @override
  State<SampleAppPage> createState() => _SampleAppPageState();
}

class _SampleAppPageState extends State<SampleAppPage> {
  List<dynamic> data = <dynamic>[];

  @override
  void initState() {
    super.initState();
    loadData();
  }

  bool get showLoadingDialog => data.isEmpty;

  Future<void> loadData() async {
    final Uri dataURL = Uri.parse(
      'https://jsonplaceholder.typicode.com/posts',
    );
    final http.Response response = await http.get(dataURL);
    setState(() {
      data = jsonDecode(response.body);
    });
  }

  Widget getBody() {
    if (showLoadingDialog) {
      return getProgressDialog();
    }
    return getListView();
  }

  Widget getProgressDialog() {
    return const Center(child: CircularProgressIndicator());
  }

  ListView getListView() {
    return ListView.builder(
      itemCount: data.length,
      itemBuilder: (context, index) {
        return getRow(index);
      },
    );
  }

  Widget getRow(int index) {
    return Padding(
      padding: const EdgeInsets.all(10),
      child: Text('Row ${data[index]['title']}'),
    );
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text('Sample App')),
      body: getBody(),
    );
  }
}
```

**Comment by huycozy** [comment]
Hi @Paul-N 
I think this is a minor fix and similar to #8920. Would you mind closing this and updating #8920's description?

**Comment by Paul-N** [comment]
Closing this. Check #8920 for details
